indico/htdocs/js/indico/jquery/global.js

```diff
@@ -166,8 +166,7 @@ $(document).ready(function() {
     });
 
     $('.js-dropdown').each(function() {
-        this.id = this.id || uniqueId();
```

Comment:
I don't think this change is correct. IIRC. the reason for using the ID or assigning a unique ID was to allow more than one dropdown within the same parent.